src/components/layout.rs

```diff
@@ -20,7 +20,7 @@ pub fn Layout<'a, G: Html>(
                 SearchBar (input=input, results=results)
             }
         }
-        main(class = "p-4 min-h-screen") {
+        main(class = "p-0 min-h-screen bg-base-200") {
```

Comment:
The padding adds some spacing around the body (we don't want the body to start right at the edge). You've also made the color of the body exactly the same as the header, which removes some depth (in my opinion).

I would suggest reverting this.

Reply:
The p-0 is to eliminate the white space between the header and the purple "search results for xxx" part, (which is also used in uwflow & GitHub etc) and the base color help contrasts the shadows of the table and filter. Also, I think it's better to use bg-base-200 as the background color cause it's not as bright as pure white and causes less eye strain.

here is "p-4"
![image](https://github.com/GauravTalreja/ebb/assets/94013624/ea4666e7-1cea-4ee6-89d9-550271519a57)
here is "p-0 bg-base-200"
![image](https://github.com/GauravTalreja/ebb/assets/94013624/d87f7a65-0225-48f6-ad70-5bab07cd0816)

Reply:
Cool, we can remove the padding!

As for the colors, this is only the case in light theme (which is auto selected because your computer is in light theme) I'm going to add the theme selector soon, which will let us test different themes. In dark mode, you are actually making the background brighter. The whole point of light mode is bright white backgrounds. If that strains a user's eye, they can switch to dark mode.

But it looks like the header is ``bg-base-300`` and the table/filter background is ``bg-base-100``so this looks good for now, we can check how it looks with different themes and decide then.